source/standard_scores.Rmd

```diff
@@ -0,0 +1,1515 @@
+reading off the y-value at which this track crosses that line.
+:::
+
+
+## Unusual values compared to the distribution
+
+Now we return the problem of whether KOMs and AOCs districts are unusual in
+terms of their median household incomes.  From what we have so far, we might
+conclude that AOC's district is fairly poor, and KOM's district is relatively
+wealthy.  But — are either of their districts *unusual* in their wealth or
+poverty?
+
+To answer that question, we have to think about the *distribution* of values.
+Are either AOC's or KOM's district outside the *typical spread* of values for
+districts?
+
+The rest of this section is an attempt to answer what we could mean by
+*outside* and *typical spread*.
+
+Let us start with a histogram of the district incomes, marking the position of
+the KOM and AOC districts.
+
+```{python echo=FALSE, eval=TRUE}
+kom_income = get_var('kom_income')
+aoc_income = get_var('aoc_income')
+plt.hist(incomes, bins=25, color='gray');
+plt.title('Median household income for 2023 congressional districts');
+plt.axvline(kom_income, color='red', linestyle=':', label='KOM')
+plt.axvline(aoc_income, color='blue', linestyle=':', label='AOC')
+plt.legend();
+```
+
+What could we mean by "outside" the "typical spread".  By outside, we mean
+somewhere away from the center of the distribution.  Let us take the mean of
+the distribution to be its center, and add that to the plot.
+
+```{python}
+mean_income = np.mean(incomes)
+```
+
+```{r}
+mean_income <- mean(incomes)
+```
+
+```{python echo=FALSE, eval=TRUE}
+mean_income = get_var('mean_income')
+plt.hist(incomes, bins=25, color='gray');
+plt.title('Median household income for 2023 congressional districts');
+plt.axvline(kom_income, color='red', linestyle=':', label='KOM')
+plt.axvline(aoc_income, color='blue', linestyle=':', label='KOM')
+plt.axvline(mean_income, color='green', linestyle='-', label='Mean')
+plt.legend();
+```
+
+## On deviations
+
+Now let us ask what we could mean by *typical spread*.  By *spread* we mean
+deviation either side of the center.
+
+We can calculate how far away each income is away from the mean, by subtracting
+the mean from all the income values.  Call the result — the *deviations* from
+the mean, or *deviations* for short.
+
+```{python}
+deviations = incomes - np.mean(incomes)
+```
+
+```{r}
+deviations <- incomes - mean(incomes)
+```
+
+The deviation values give, for each district, how far that district's income is
+from the mean.  Values near the mean will have small (positive or negative)
+values, and values further from the mean will have large (positive and
+negative) values.  Here is a histogram of the *deviation* values.
+
+```{python echo=FALSE, eval=TRUE}
+deviations = get_var('deviations')
+plt.hist(deviations, bins=25, color='gray');
+plt.title('Deviations of median household income for 2023 congressional districts');
+```
+
+The distribution of deviations centers on zero — an in fact the deviations have
```

Comment:
This is the only difference. Note that the *shapes* of the two plots are identical.